lib/t_web/channels/feed_channel.ex

```diff
@@ -478,12 +478,12 @@ defmodule TWeb.FeedChannel do
     {:noreply, socket}
   end
 
-  def handle_info({Matches, [:timeslot, :start], match_id}, socket) do
+  def handle_info({Matches, [:timeslot, :started], match_id}, socket) do
     push(socket, "timeslot_started", %{"match_id" => match_id})
     {:noreply, socket}
   end
 
-  def handle_info({Matches, [:timeslot, :end], match_id}, socket) do
+  def handle_info({Matches, [:timeslot, :ended], match_id}, socket) do
```

Comment:
and this are the only places that needed to change

Reply:
if possible, please remove the commits from this pr and reapply them in a separate pr targeting master, it's confusing now

Reply:
sure